modules/optimization/examples/bimaterial/model.i

```diff
@@ -0,0 +1,182 @@
+# Heat conduction in a 2D domain with two diffusivities
+# The domain is -5 <= x <= 5 and -5 <= y <= 5
+# The top-half of the domain (y > 0) has high diffusivity
+# The top-half of the domain (y < 0) has low diffusivity
+# Temperature is initialised to zero
+# Temperature is fixed at temperature = 1 at centre of the domain (which is on the interface between the top and bottom halves)
+#
+# Assume experimental observations of the temperature are made at time = 1, and yield:
+# at point = (-2, -2, 0), temperature = 0.022
+# at point = (0, -2, 0), temperature = 0.040
+# at point = (2, -2, 0), temperature = 0.022
+# at point = (0, 2, 0), temperature = 0.138
+# Notice that the temperature is higherin the top (y > 0) half of the domain because the diffusivity is higher there
+#
+# This MOOSE model will give different predictions for temperature, depending on the time-step size and the spatial resolution
+# For dt = 1, nx = 100, ny = 100, the experimental observations are well-matched for
+# diffusivity of top ~ 10
+# diffusivity of bottom ~ 1
+#
+# The purpose of the main.i input file is to find this solution
+#
+# The unusual things about this input file are:
+# - the diffusivity values are not just stored in two Materials.  Instead, the Materials get their values from Functions (they are GenericFunctionMaterials) which get their values from Postprocessors, which get their values from a VectorPostprocessor.  The reason for this rather convoluted approach is because the VectorPostprocessor can be controlled by a MultiAppReporterTransfer in main.i
```

Comment:
```suggestion
# - the diffusivity values are not just stored in two Materials.  Instead, the Materials get their values from Functions (they are GenericFunctionMaterials)
#   which get their values from Postprocessors, which get their values from a VectorPostprocessor.  The reason for this rather convoluted approach is 
#   because the VectorPostprocessor can be controlled by a MultiAppReporterTransfer in main.i
```

Reply:
removed this example